### PR TITLE
Add Support for multiple TXT Records. Fixes #4.

### LINKED ISSUE #4: Support for multiple character strings in TXT records
In the RFC 1035 ([https://www.ietf.org/rfc/rfc1035.txt](url)) it is specified that TXT-DATA is "[o]ne or more <character-string>s", a <character-string> being described as "a single length octet followed by that number of characters". The implementation of the ReadString() method called by the RecordTXT constructor, however, reads only the initial octet and thus returns a record with only the first <character-string> from records with multiple <character-string>s, and thus answers to TXT queries only contain the first <character-string>, contrary to the specification in RFC 1035.

I have seen various behaviors in resolvers for dealing with multiple <character-string>s. For instance, dig returns each string wrapped in quotation marks, delimited by a space, while libresolv returns the byte array with the octets indicating length in place. I am relatively new to C# and .NET generally so I am unsure as to the best way to reimplement RecordTXT to support multiple <character-string>s.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghuntley** [comment]
@jmdevince  do you want to take over maintenance of this library?

**Comment by ghost** [comment]
Sure. I use the library and I'm fluent in C#.

**Comment by ghuntley** [comment]
Welcome aboard, check your email for an invitation.

File: src/Heijden.Dns/Records/RecordTXT.cs [hunk 1]
```diff
@@ -22,17 +24,22 @@ namespace Heijden.DNS
 {
 	public class RecordTXT : Record
 	{
-		public string TXT;
+		public List<string> TXT;
```
Comment (ghuntley) [inline src/Heijden.Dns/Records/RecordTXT.cs:27]: This is a breaking change; we will need to bump the SemVer to 2.0

**Comment by ghuntley** [comment]
v2.0.0.x has been pushed to NuGet.org

**Pull request merged**